src/javascripts/ng-admin/Crud/button/maExportToCsvButton.js

```diff
@@ -4,8 +4,7 @@ export default function maExportToCsvButton ($stateParams, Papa, notification, A
         scope: {
             entity: '&',
             label: '@',
-            datastore: '&',
-            search: '&'
```

Comment:
You'll have to remove the param where the directive is called, in `listLayout.thml`